code/modules/mob/living/simple_animal/hostile/changeling.dm

```diff
@@ -139,3 +139,94 @@
 	var/obj/item/bone_dart/A = new /obj/item/bone_dart(usr.loc)
 	A.throw_at(target, 10, 20, usr)
 	add_logs(src, target, "launched a bone dart at")
+
+/mob/living/simple_animal/hostile/lesser_changeling
+	name = "crawling horror"
+	desc = "An agile monster made of twisted flesh and bone.."
+	speak_emote = list("gibbers")
+	icon = 'icons/mob/npc/animal.dmi'
+	icon_state = "lesser_ling"
+	icon_living = "lesser_ling"
+
+	stop_automated_movement = TRUE
+	universal_speak = TRUE
+	universal_understand = TRUE
+
+	tameable = FALSE
+
+	response_help  = "pets"
+	response_disarm = "shoves"
+	response_harm   = "harmlessly punches"
```

Comment:
How is it gonna punch with that body bro

Reply:
That is when you punch it.